function/.eslintrc.json

```diff
@@ -1,5 +1,12 @@
 {"extends": "strongloop",
  "parserOptions": {
-   "ecmaVersion": 6
+   "ecmaVersion": 2017
+ },
```

Comment:
`"ecmaVersion": 2017` is necessary because we have `async` flags in some tests - with `"ecmaVersion": 6`, lint has compile/parse errors. I'm not super pleased about this situation because node 6 - which is our Cloud Function runtime - doesn't support ecma 2017.